src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -82,9 +104,14 @@ object Lorre extends App with TezosErrors with LazyLogging {
       else processing
 
     Await.result(processResult, atMost = Duration.Inf)
-    logger.info("Taking a nap")
-    Thread.sleep(sleepIntervalInSeconds * 1000)
-    mainLoop(iteration + 1)
+
+    depth match {
+      case Newest =>
+        logger.info("Taking a nap")
+        Thread.sleep(sleepIntervalInSeconds * 1000)
+        mainLoop(iteration + 1)
+      case _ => ()
```

Comment:
Maybe for this case we can leave a log message saying something like "My work is done."?